_sass/minima/_layout.scss

```diff
@@ -406,4 +406,13 @@ div.BAN-list {
   justify-content: center;
 }
 
+/*
```

Comment:
Might want to eventually make a _include.scss file for css specifically for different include components?